site/static/css/styles.css

```diff
@@ -3114,9 +3114,11 @@ a:hover {
 
 code {
     word-break: break-word;
+    font-family: 'Fira Mono','Courier New',Courier,monospace;
 }
 
-td:first-child code {
+/* Ensure words are not broken for monospaced text in a table cell */
+td>code {
```

Comment:
Is this change for functionality or for readability? This changes how the word wrap works for columns other than just the first column of a table. The original intention was to only affect the first column. This has a large impact on all of the docs (not just Apps-related docs), so I want to make sure this is what we want.

Reply:
It impacts any code blocks in a table cell.
We don't want our table data to break across lines if, say, it's an object or function name; readability would be affected. I haven't yet seen an example where this CSS change has broken existing tables.